robosuite/utils/mjcf_utils.py

```diff
@@ -95,34 +95,35 @@
     "opencv": -1,
 }
 
+DEFAULT_TEXTURE_PATH = "textures/"
 TEXTURES = {
-    "WoodRed": "red-wood.png",
-    "WoodGreen": "green-wood.png",
-    "WoodBlue": "blue-wood.png",
-    "WoodLight": "light-wood.png",
-    "WoodDark": "dark-wood.png",
-    "WoodTiles": "wood-tiles.png",
-    "WoodPanels": "wood-varnished-panels.png",
-    "WoodgrainGray": "gray-woodgrain.png",
-    "PlasterCream": "cream-plaster.png",
-    "PlasterPink": "pink-plaster.png",
-    "PlasterYellow": "yellow-plaster.png",
-    "PlasterGray": "gray-plaster.png",
-    "PlasterWhite": "white-plaster.png",
-    "BricksWhite": "white-bricks.png",
-    "Metal": "metal.png",
-    "SteelBrushed": "steel-brushed.png",
-    "SteelScratched": "steel-scratched.png",
-    "Brass": "brass-ambra.png",
-    "Bread": "bread.png",
-    "Can": "can.png",
-    "Ceramic": "ceramic.png",
-    "Cereal": "cereal.png",
-    "Clay": "clay.png",
-    "Dirt": "dirt.png",
-    "Glass": "glass.png",
-    "FeltGray": "gray-felt.png",
-    "Lemon": "lemon.png",
+    "WoodRed": DEFAULT_TEXTURE_PATH + "red-wood.png",
```

Comment:
Please use `os.path.join` to concatenate file paths. simple string addition is error-prone, as it requires the "/" as the end of the folder path.

Reply:
I've changed the concatenating operation to using `os.path.join`.